### ISSUE TITLE
Run experiments with stochastic gradient boosting

### ISSUE THREAD

**Issue by jerradmgenson** [jerradmgenson/qdaim-proto#51]
Stochastic gradient boosting is a similar method to random forest for generating models that may offer improved accuracy. Let's investigate it on our dataset to see how the performance of this method compares.

**Comment by jerradmgenson** [comment]
Scikit-learn has a classifier for this here: https://scikit-learn.org/stable/modules/generated/sklearn.ensemble.GradientBoostingClassifier.html#sklearn.ensemble.GradientBoostingClassifier

**Comment by jerradmgenson** [comment]
Duplicates issue #62
